step.yml

```diff
@@ -91,3 +113,9 @@ outputs:
 - BITRISE_APK_PATH:
   opts:
     title: "Bitrise signed build artifact path"
+- BITRISE_SIGNED_AAB_PATH:
+  opts:
+    title: "Bitrise signed build artifact path"
+- BITRISE_AAB_PATH:
+  opts:
+    title: "Bitrise signed build artifact path"
```

Comment:
explicitly mention that this output stores only Android App Bundle files and also pls update the APK path related outputs as well